Utils/utils/password_generator.py

```diff
@@ -0,0 +1,30 @@
+import secrets
+import string
+import random
+from sekoia_automation.action import Action
+
+
+class PasswordGenerator(Action):
+    def run(self, arguments: dict):
+        password_length = arguments.get("password_length", 20)
+        number_of_digits = arguments.get("number_of_digits", 1)
+        number_of_special_characters = arguments.get("number_of_special_characters", 1)
+
+        if number_of_digits + number_of_special_characters > password_length:
+            raise ValueError("number_of_digits + number_of_special_characters must be lower than password_length")
```

Comment:
Raising an exception will result in having the errors message `An unexpected error occured: ...`.

```suggestion
            self.error("number_of_digits + number_of_special_characters must be lower than password_length")
            return
```